esrally/rally.py

```diff
@@ -191,7 +191,7 @@ def positive_number(v):
         track_source_group.add_argument(
             "--track-path",
             help="Define the path to a track.")
-        track_source_group.add_argument(
+        p.add_argument(
```

Comment:
Could we add an integration test?

We could use the [list command](https://github.com/elastic/rally/blob/966891a38397227300d55ac32581e4b1d9d53d85/integration-test.sh#L219-L233): https://github.com/elastic/rally/blob/966891a38397227300d55ac32581e4b1d9d53d85/integration-test.sh#L219-L233

and, in order to avoid having to add the `eventdata` repo in the rally.ini used by the integration script (which is randomized btw), we could just rely on the default repository with something like:

```
   info "test list can use track revision together with track repository"
    esrally list tracks --configuration-name="${cfg}" --track-repository=default --track-revision=4080dc9850d07e23b6fc7cfcdc7cf57b14e5168d
}
```